MANIFEST.in

```diff
@@ -0,0 +1,8 @@
+include README.md LICENSE settings.yml
+recursive-include buildtest *
+recursive-include buildtest *.yml
```

Comment:
i am a bit confused with these line. Instead of ``recursive-include buildtest *`` just use ``graft buildtest``. That should get all source code under ``buildtest`` directory which is required for packaging.

Reply:
You're right this is a bit messy - I haven't used graft before I'll give that a try now!